arch/arc/Kconfig

```diff
@@ -135,9 +135,59 @@ config CODE_DENSITY
 config ARC_HAS_SECURE
 	bool
 	#  a hidden option
+	select CPU_HAS_TEE
+	select ARCH_HAS_TRUSTED_EXECUTION
 	help
 	  This option is enabled when ARC core supports secure mode
 
+config SJLI_TABLE_SIZE
+	int "SJLI table size"
+	depends on ARC_SECURE_FIRMWARE
+	default 8
+	help
+	   The size of sjli (Secure Jump and Linux Indexed) table. The
```

Comment:
interesting typo, can jump to Linux with a sjli call ;-)

Reply:
fixed